.travis.yml

```diff
@@ -1,14 +1,14 @@
 language: go
 go_import_path: github.com/atlassian/kubetoken
 go:
-  - 1.8.x
+  - 1.9.x
```

Comment:
Why not 1.10.x?

Reply:
Originally just bumped the minimum version from 1.8 -> 1.9 as 1.8 doesn't have windows syscall.Filetime, but I've added 1.10